lit_gpt/model.py

```diff
@@ -158,15 +157,15 @@ def forward(
         h = self.attn(n_1, cos, sin, mask, input_pos)
         if self.config.parallel_residual:
             n_2 = n_1 if self.config.shared_attention_norm else self.norm_2(x)
-            x = x + h + self.mlp(n_2)
+            x = self.mlp(n_2) + h + x
```

Comment:
Addition is not commutative in fp16, and this is how GptNeox implements the order